packages/contracts-bedrock/scripts/checks/check-frozen-files.sh

```diff
@@ -52,10 +52,10 @@ ALLOWED_FILES=(
   # "src/L1/L1CrossDomainMessenger.sol"
   # "src/L1/L1ERC721Bridge.sol"
   # "src/L1/L1StandardBridge.sol"
-  # "src/L1/OPContractsManager.sol"
-  # "src/L1/OPContractsManagerInterop.sol"
-  # "src/L1/OptimismPortal2.sol"
-  # "src/L1/OptimismPortalInterop.sol"
+  "src/L1/OPContractsManager.sol"
+  "src/L1/OPContractsManagerInterop.sol"
+  "src/L1/OptimismPortal2.sol"
+  "src/L1/OptimismPortalInterop.sol"
```

Comment:
Why are we allowing changes to these files? My understanding is we need to wait until after the OPCM Upgrade 13 release is cut to merge this, because we are not shipping this change in upgrade 13?

Reply:
Merging into a feature branch

Reply:
Check, maybe doesn't apply to feature branches